packages/hash/frontend/src/pages/[account-slug]/types/entity-type/use-property-types.tsx

```diff
@@ -2,44 +2,46 @@ import { PropertyType, VersionedUri } from "@blockprotocol/type-system-web";
 import { createContext, useContext, useEffect, useState } from "react";
 import { useBlockProtocolAggregatePropertyTypes } from "../../../../components/hooks/blockProtocolFunctions/ontology/useBlockProtocolAggregatePropertyTypes";
 import { mustBeVersionedUri } from "./util";
+import { getPersistedPropertyType } from "../../../../lib/subgraph";
+import { useAdvancedInitTypeSystem } from "../../../../lib/use-init-type-system";
 
 type PropertyTypesContextValues = Record<VersionedUri, PropertyType>;
 
 export const useRemotePropertyTypes = () => {
+  const [typeSystemLoading, _] = useAdvancedInitTypeSystem();
+
   const [propertyTypes, setPropertyTypes] =
     useState<PropertyTypesContextValues | null>(null);
   const { aggregatePropertyTypes } = useBlockProtocolAggregatePropertyTypes();
 
   useEffect(() => {
-    void aggregatePropertyTypes({ data: {} }).then(({ data }) => {
+    if (typeSystemLoading) {
```

Comment:
@nathggns I've had to add this check, is that alright? 

I thought we were going to wrap all of the pages in these but it seems we're calling them in individual places?

Reply:
I think we're doing this on an individual page level still – but i've changed this in a follow up PR to share the status by context. This seems a sensible change